### PR TITLE
Add bnode Deduplication enrichment

### PR DESCRIPTION
**Pull request by no-reply**

### CODE REVIEW

**Comment by no-reply** [comment]
This should be in for ESDN enrichment.

File: lib/krikri/enrichments/dedup_nodes.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  ##
+  # Enrichment to remove duplicate blank nodes from a Graph, where "duplicate"
+  # means having the same `providedLabel`.
+  class DedupNodes
+    include Krikri::FieldEnrichment
+
+    ##
+    # @param value [Object]
+    #
+    # @return [Object] the original value altered to remove nodes with the same
```
Comment (anarchivist) [inline lib/krikri/enrichments/dedup_nodes.rb]: add an `@example`?

**Comment by anarchivist** [comment]
I'm :+1: if we can improve documentation with an `@example`.

**Comment by no-reply** [comment]
:+1: will do.

**Comment by no-reply** [comment]
Doc added and squashed.

**Pull request merged**